tests/base_tests.py

```diff
@@ -816,3 +818,12 @@ class OtherBaseBehaviourTests(
     Class to collect and hold tests for BaseTransformerBehaviour outside the three standard methods.
     Note this deliberately avoids starting with "Tests" so that the tests are not run on import.
     """
+
+    def test_is_serialisable(self, initialized_transformers):
+        path = "transformer.pkl"
```

Comment:
I do think the solution of using the pytest tmp path is neater if we can get it working. I looked at the past commits you made and it looks like you used tmpdir instead of tmp_path. Wondering if you had tried both?

SO thread here discussing the differences - https://stackoverflow.com/questions/36070031/creating-a-temporary-directory-in-pytest

Reply:
good call, tmp_path worked well - [commit](https://github.com/lvgig/tubular/pull/258/commits/cf55df03b931dd05e5307d5fdfee870ad0da6d05)

Reply:
Nice, thanks :)